packages/dashboard/src/components/querySettingsSync/utils.ts

```diff
@@ -0,0 +1,14 @@
+export const selectIntialRefreshRate = (refreshRate?: number) => {
```

Comment:
The name for this function `select*` sounds like it is mutating and initializing something. It's more it should be called `createRefreshRateOption`.